core/admin/mailu/configuration.py

```diff
@@ -40,9 +40,9 @@
     'TLS_FLAVOR': 'cert',
     'INBOUND_TLS_ENFORCE': False,
     'DEFER_ON_TLS_ERROR': True,
-    'AUTH_RATELIMIT_IP': '60/hour',
+    'AUTH_RATELIMIT_IP': '5/hour',
     'AUTH_RATELIMIT_IP_V4_MASK': 24,
-    'AUTH_RATELIMIT_IP_V6_MASK': 56,
+    'AUTH_RATELIMIT_IP_V6_MASK': 48,
```

Comment:
You need to alter the comment in the mailu.env file to `/48` if you change this.
I'm not sure if blocking a /48 is a good idea. At least her in germany many provider give you the bare minimum of addresses (/64 or /60)
And I know of hosting setups using a single /64 for hundreds of servers from differnet customers.

Reply:
done in 83cc23a5